migrations/2025-09-08-140711_remove-actor-name-max-length/up.sql

```diff
@@ -0,0 +1,46 @@
+-- get rid of max name length setting
+ALTER TABLE local_site
+    DROP COLUMN actor_name_max_length;
+
+-- truncate existing strings
+UPDATE
+    person
+SET
+    name = substring(name FROM 1 FOR 20)
+WHERE
+    length(name) > 20;
+
+UPDATE
+    person
+SET
+    display_name = substring (display_name FROM 1 FOR 20)
+WHERE
+    length(display_name) > 20;
+
+UPDATE
+    community
+SET
+    name = substring(name FROM 1 FOR 20)
+WHERE
+    length(name) > 20;
+
+UPDATE
+  community
+SET
+    title = substring (title FROM 1 FOR 20)
+WHERE
+    length(title) > 20;
```

Comment:
The name changes will fail if there is already a user/community with this name. Any way to avoid that?

Also this can cause problems with federation. If the name of a local person or community is changed like this, other instances will have it both under the old and the new name. In case of remote person/community we cant fetch content or send activities to it as the name is wrong. But thats probably unavoidable.